app/controllers/responsible_body/devices/who_to_contact_controller.rb

```diff
@@ -18,8 +18,8 @@ def create
       render :new, status: :unprocessable_entity
     else
       chosen_contact = @form.chosen_contact
-      chosen_contact.save! unless chosen_contact.persisted?
```

Comment:
i'm not sure why this guard is here. is it to guard some callback or notification?

Reply:
I don't think there is a really good reason for it being here. I probably added to avoid saving the headteacher contact again but then if it's already saved, the operation is a noop...

I think it's fine to remove.